server/app/repository/ApplicationStatusesRepository.java

```diff
@@ -52,4 +54,26 @@ public List<StatusDefinitions> lookupListOfObsoleteStatusDefinitions(String prog
         .map(ApplicationStatusesModel::getStatusDefinitions)
         .collect(Collectors.toList());
   }
+
+  public void updateStatusDefinitions(String programName, StatusDefinitions statusDefinitions) {
```

Comment:
**suggestion, optional, not blocking**: Depending on how we're calling this, a single `createOrUpdateStatusDefinitions()` method could make the callsites cleaner.

Reply:
Create has to be different as it would have the update() query.

Reply:
Yeah, inside the method it would have check if the statuses existed for the program and, if they did, call update and if they didn't call create.

_very_ optional thought though, I don't know what your callsites for this look like so I don't know if it's easier to hide that complexity in the Repository or let it be at the callsite.